understanding/name-role-value.html

```diff
@@ -58,106 +56,79 @@
-                  information
-                  to assistive technologies and allow themselves to be controlled by assistive technologies.
+               <p>しかし、カスタムコントロールが作成される、又はインタフェース要素が通常とは異なる役割及び／又は機能を持つように (ソースコード又はスクリプトで) プログラミングされている場合、コントロールが支援技術に重要及び適切な情報を提供し、支援技術によってコントロールされることを、それら自身ができるように保証する追加の手段を取る必要がある。
                   
                </p>
-               <p>What roles and states are appropriate to convey to assistive technology will depend
-                  on what the control represents. Specifics about such information are defined by other
-                  specifications, such as <a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a>, or the
-                  relevant platform standards. Another factor to consider is whether there is sufficient
-                  <a href="https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported">accessibility support</a> 
-                  with assistive technologies to convey the information as specified.
+               <p>どのような役割 (role) 及び状態を支援技術に伝えるのが適切かは、コントロールが何を表すかによる。そのような情報に関する詳細は、<a href="https://www.w3.org/TR/wai-aria/">WAI-ARIA</a> などの他の仕様、関連するプラットフォーム標準によって定義される。考慮すべきもう一つの要素は、指定されたとおりに情報を伝達するために、支援技術とともに十分な<a href="https://waic.jp/translations/WCAG22/#dfn-accessibility-supported">アクセシビリティサポート</a>があるかどうかである。
                   
                </p>
-               <p>A particularly important state of a user interface control is whether or not it has
-                  focus. The focus state of a control can be programmatically determined, and notifications
-                  about change of focus are sent to user agents and assistive technology. Other examples
-                  of user interface control state are whether or not a checkbox or radio button has
-                  been selected, or whether or not a collapsible tree or list node is expanded or collapsed.
+               <p>特に重要なユーザインタフェース コントロールの状態は、フォーカスを持つかどうかである。コントロールのフォーカス状態は、プログラムによる解釈が可能であり、フォーカスの変化に関する通知は、ユーザエージェント及び支援技術に送られる。ユーザインタフェース コントロールのステータスの他の例としては、チェックボックス又はラジオボタンが選択されているかどうか、折り畳み可能なツリーやリストが展開されている又は折り畳まれているかどうかである。
                   
                </p>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      
                      
-                     <p>Success Criterion 4.1.2 requires a programmatically determinable name for all user
-                        interface components. Names may be visible or invisible. Occasionally, the name needs
-                        to be visible, in which case it is identified as a label. Refer to the definition
-                        of
-                        name and label in the glossary for more information.
+                     <p>達成基準 4.1.2 は、すべてのユーザインタフェース コンポーネントに対して、プログラムによる解釈が可能な名前を要求する。名前は、可視であっても、不可視であってもよい。往々にして、名前がラベルとして識別される場合、名前は可視である必要がある。より詳しい情報については、用語集にある名前及びラベルの定義を参照のこと。
```

Comment:
「往々にして、名前がラベルとして識別される場合、名前は可視である必要がある。」は誤訳のように思います。ここでは「名前が可視だったらそれはラベルにも該当する」というような話のはずです。こんな感じでしょうか。

```suggestion
                     <p>達成基準 4.1.2 は、すべてのユーザインタフェース コンポーネントに対して、プログラムによる解釈が可能な名前を要求する。名前は、可視であっても、不可視であってもよい。往々にして、名前は可視にする必要があり、その場合はラベルとして識別される。より詳しい情報については、用語集にある名前及びラベルの定義を参照のこと。
```